content/blog/2016/2016-04-07-jenkins-community-survey-results-blog.adoc

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Jenkins Community Survey Results: CD, Pipelines, and Star Wars"
+tags:
+- jenkins
+- survey
+- continuous delivery
+- pipelines
+- docker
+- star wars
```

Comment:
I suggest the tags:

```yaml
- continuousdelivery
- pipeline
- docker
```

These are primarily used as post-metadata for the site generation system, so a more concise list is better